### DISCUSSION TITLE
Unlocker says game is running when it isn't

### DISCUSSION THREAD

**Discussion by Deztacular** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#7] [Q&A]
The unlocker is saying to close the game before running but no instances are showing in task manager.

**Comment by WakuWakuPadoru** [comment]
> The unlocker is saying to close the game before running but no instances are showing in task manager.

The launcher is included as well. Check your system tray in your taskbar and check that the launcher is not still open.

**Reply by WakuWakuPadoru** [reply]
The Unlocker specifically checks for "launcher.exe" and "Wuthering Waves.exe" if you need to close them manually in the task manager. Reason for this "check" is to minimise the possibility of having multiple applications access the same config file.
